SC_redesign/Crop_and_Soil/crop/crop.py

```diff
@@ -56,26 +57,25 @@ def __init__(self, crop_data: Optional[CropData] = None):
         #  composite design would require interdependence such that these
         #  common attributes remain in-sync. - GitHub Issue #255
 
-
     def grow_crop(self, layer_nitrates: List[float], layer_depths: List[float],
                   layer_phosphates: List[float],
                   soil_water_factor: float,
-                  max_transpiration: float, air_temperature: float,
+                  max_transpiration: float,
                   incoming_light: float,
                   evaporation: float, transpiration: float,
                   max_evapotranspiration: float,
                   mean_air_temperature: float, min_air_temperature: float,
-                  max_air_temperature: float) -> None:
+                  max_air_temperature: float, adjusted_potential_evapotranspiration: float) -> None:
         """main function for growing the crop on a daily basis
 
         Args:
+            adjusted_potential_evapotranspiration:
```

Comment:
```suggestion
            adjusted_potential_evapotranspiration: potential evapotranspiration adjusted for evaporation of free water in canopy in mm
```
This is how it is currently defined in `soil\evapotranspiration.py`.